apps/angular/src/app/features/auth/auth-routing.module.ts

```diff
@@ -0,0 +1,28 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+import { LoginComponent } from './login/login.component';
+import { RegistrationComponent } from './registration/registration.component';
+
+const routes: Routes = [
+	{
+		path: '',
+		redirectTo: '/auth/login',
```

Comment:
If I'm not mistaken 'redirectTo' will work in the scope of current routing module. So we'd rather do something like this:
```suggestion
		redirectTo: 'login',
```